.github/workflows/comment-diff.yaml

```diff
@@ -7,6 +7,13 @@ on:
 permissions: read-all
 
 jobs:
+  dump_contexts_to_log:
```

Comment:
Any possibility of dumping secrets or other sensitive info when doing this?

Reply:
No. GitHub knows to mask its own tokens. I've run this on my own fork and it was masked: https://github.com/zkamvar/samehcs/actions/runs/11258438269/job/31305034285

I'm going to remove it once I have debugged.